version.php

```diff
@@ -24,7 +24,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2019061200;    // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2019062600;    // The current plugin version (Date: YYYYMMDDXX).
 $plugin->release   = 2019061200;    // Match release exactly to version.
```

Comment:
Also can you please bump $plugin-release in lockstop with plugin->version

Reply:
whoops - missed you were doing that.